src/v2_services/chat_service.py

```
@@ -28,16 +28,18 @@
     → 5단계: LLM(Gemini)이 근거 자료 기반으로 답변 생성
     → 6단계: API 응답 반환
 """
-
-from typing import Dict, Any, List
+import json
+from typing import Dict, Any, List, Optional
```

Ruff (0.15.5)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)